sechub-server/src/main/resources/db/migration/common/U29__encryption.sql

```diff
@@ -0,0 +1,17 @@
+-- SPDX-License-Identifier: MIT
+
+-- encryption parts
+-- FIXME de-jcup: discussion necessary, maybe server 2.0.0 (major update necessary)
```

Comment:
I thought about it and the downgrade SQL scripts will be implemented/available, but with a remark that it is not recommended to use them because of potential data loss.